### ISSUE TITLE
Writeback cache flushes data too frequently

### ISSUE THREAD

**Issue by hipporoll** [stec-inc/EnhanceIO#37]
Hi, there's another issue about writeback cache.
What I want to set up is an usual writeback cache that flushes only once in a while, let's say once in 5 minutes. For that purpose I set time_based_clean_interval to 5.

Setup command:
eio_cli create -d /dev/sdc -s /dev/md0 -p lru -m wb -c CACHE -b 8192

Problem is that cache flushes data into the source device very frequently, nearly continuously even in the first 10 seconds. Practically I don't have to wait for 5 minutes until cache flush happens. In this case, tuning time_based_clean_interval doesn't bring anything.

As long as I've analyzed so far, eio_write() seems to call eio_uncached_write() nearly always, which might trigger the so many flushes. Is there any way to avoid uncached writes, for normal sync write I/O at least?

Dongsu
